src/hooks/useEnableToken.ts

```diff
@@ -29,29 +29,40 @@ export const useEnableTokens = (params: Params): Result => {
     }
   }, [])
 
-  async function enableToken(): Promise<TxResult<boolean>> {
+  async function enableToken(): Promise<TxResult<boolean> | boolean> {
```

Comment:
Why this complex signature, either is a reject, or resolves giving you the receipt (well the receipt is in another branch out there, here is a TxResult)